scripts/add_electricity.py

```diff
@@ -406,18 +430,32 @@ def attach_wind_and_solar(
                         * costs.at[car + "-connection-underground", "capital_cost"]
                     )
                 )
+                connection_investment = (
+                    line_length_factor
+                    * ds["average_distance"].to_pandas()
+                    * (
+                        underwater_fraction
+                        * costs.at[car + "-connection-submarine", "investment"]
+                        + (1.0 - underwater_fraction)
+                        * costs.at[car + "-connection-underground", "investment"]
+                    )
+                )
                 capital_cost = (
                     costs.at["offwind", "capital_cost"]
                     + costs.at[car + "-station", "capital_cost"]
                     + connection_cost
                 )
+                investment = costs.at["offwind", "investment"]
+                connection_investment += costs.at[car + "-station", "investment"]
                 logger.info(
                     "Added connection cost of {:0.0f}-{:0.0f} Eur/MW/a to {}".format(
                         connection_cost.min(), connection_cost.max(), car
                     )
                 )
             else:
                 capital_cost = costs.at[car, "capital_cost"]
+                investment = costs.at[car, "investment"]
+                connection_investment = None
```

Comment:
Perhaps 0 would be a sensible default value or `pd.NA/np.nan`. `None` is not a good choice I think.